crates/e2e/tests/e2e/order_cancellation.rs

```diff
@@ -214,10 +214,10 @@ async fn order_cancellation(web3: Web3) {
             .status,
         OrderStatus::Cancelled,
     );
-    assert_eq!(
-        services.get_order_status(&order_uids[0]).await.unwrap(),
-        orderbook::dto::order::Status::Cancelled,
-    );
+    let events = crate::database::events_of_order(services.db(), &order_uids[0]).await;
+    assert!(events
```

Comment:
nit: maybe comment on the counterintuitive use of `any` (due to the soft nature of offchain cancellations, it's possible that an order gets scheduled, ie ready, after being cancelled)